academy/xl-cosmjs/with-kepler.md

```diff
@@ -182,7 +182,7 @@ Following [Keplr's documentation](https://docs.keplr.app/api/#how-to-detect-kepl
 $ npm install @keplr-wallet/types --save-dev
 ```
 
-And inform Typescript that `window` may have a `.keplr` field with the help of [this helper](https://github.com/chainapsis/keplr-wallet/tree/master/docs/api#keplr-specific-features), by adding it to `FaucetSender.tsx`:
+And inform Typescript that `Window` may have a `.keplr` field with the help of [this helper](https://github.com/chainapsis/keplr-wallet/tree/master/docs/api#keplr-specific-features), by adding it to `FaucetSender.tsx`:
```

Comment:
```suggestion
And inform Typescript that `window` may have a `.keplr` field with the help of [this helper](https://github.com/chainapsis/keplr-wallet/tree/master/docs/api#keplr-specific-features), by adding it to `FaucetSender.tsx`:
```
@AndrewB9lab Actually, it is `window` as can be seen on line 289. `window` is the instantiated object, which _may_ have a `.keplr`. `KeplrWindow` and `Window` are interface types that help describe the type of `window`.